ports/servoshell/egl/ohos.rs

```diff
@@ -126,6 +130,11 @@ static PROMPT_TOAST: OnceLock<
     ThreadsafeFunction<String, (), String, false, false, PROMPT_QUEUE_SIZE>,
 > = OnceLock::new();
 
+/// Currently we do not support different contexts for different windows but we might want to change tabs.
+/// For this we store the window context for every tab and change the compositor by hand.
+static WEBVIEW_TO_RAW_HANDLE: LazyLock<Mutex<Vec<(XComponentWrapper, WindowWrapper)>>> =
```

Comment:
Why do we need the LazyLock here? both Mutex and Vec should be able to be constructed in `const` contexts.